lib/tools/createToken.ts

```diff
@@ -0,0 +1,85 @@
+import { tool } from "ai";
+import { z } from "zod";
+import { CreateTokenResponse } from "../toolResponse.types";
+
+const createToken = (question: string) => tool({
+  description: `Create a new token.
+    IMPORTANT: Always call this tool first for ANY question related to creating token.
+    Do NOT attempt to answer questions on these topics without calling this tool first.
+    
+    Example questions that MUST trigger this tool:
+    - "Create a new token."
+    - "I wanna create a new token."`,
+  parameters: z.object({
+    address: z.string().describe("The connected coinbase smart wallet."),
+    image: z.string().optional().describe("The image to create the token with."),
+    animation: z.string().optional().describe("The media to create the token with."),
+    title: z.string().optional().describe("The title of the token."),
+    collectionAddress: z.string().optional().describe("The contract address of the collection."),
+    mimeType: z.string().optional().describe("The type of media."),
+    transaction: z.string().optional().describe("The transaction of token creation."),
```

Comment:
I'm not sure `transaction` will be triggered in a function call.

Could you please remove this `parameter` for now?